packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/props.ts

```diff
@@ -204,9 +204,9 @@ const propTypes: PropValidators<PropKeys> = {
   mountNode: PropTypes.oneOfType([element, PropTypes.func]),
   defaultFocusElement: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
   liveRegion: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.element),
-    PropTypes.element,
-    PropTypes.func
+    PropTypes.arrayOf(PropTypes.elementType),
+    PropTypes.elementType,
+    PropTypes.number
```

Comment:
This `number` type seems to be unintentional :)

Reply:
yep, fixed